### PR TITLE
Fix issues with SqlMagic (prettytable) and arrow dependency conflict (edited in Colab)

### PR DESCRIPTION
**Pull request by ColinMaudry**
Source @allakulov: [Slack](https://opencontractingint.slack.com/archives/C01H84M900H/p1732121896689579)

Same PR as https://github.com/open-contracting/notebooks-ocds/pull/74 but edited in Colab instead of locally.

### CODE REVIEW

**Comment by yolile** [comment]
@ColinMaudry I will resolve the reported lint issue here as it relates to something you haven't touched

**Comment by ColinMaudry** [comment]
Thanks!

**Comment by jpmckinney** [comment]
I think part of this needs to be fixed in kingfisher-colab instead. Its tests are failing (probably due to a updated dependency causing issues).

**Comment by jpmckinney** [comment]
Indeed, prettytable just needs to be frozen to 3.11.0 https://github.com/catherinedevlin/ipython-sql/issues/252 https://github.com/prettytable/prettytable/releases/tag/3.12.0

ipython-sql doesn't interact with prettytable through its `set_style` API and instead reads from the module's `__dict__`, which is Very Bad.

File: component_setup_kingfisher.ipynb [hunk 1]
```diff
@@ -68,6 +68,9 @@
         "# Don't show connection string after execute.\n",
         "%config SqlMagic.displaycon = False\n",
         "\n",
+        "# Set SqlMagic/prettytable style\n",
+        "%config SqlMagic.style = '_DEPRECATED_DEFAULT'\n",
+        "\n",
```
Comment (jpmckinney) [inline component_setup_kingfisher.ipynb:73]: You can remove this. Just make sure to use the latest ocdskingfishercolab (0.3.14).

**Pull request merged**